index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio Neeraj</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <header>
+      <div class="header-nav-phone">
+        <a href="/" class="header-logo">
+          Welcome
+        </a>
+        <button type="button" class="menu-btn">
+          <img src="./asset/Button.svg" alt="menu link" />
+        </button>
+      </div>
+    </header>
+    <section class="headline">
+      <div class="container container-flex">
+        <h1>
+          I’m Neeraj
+        </h1>
+        <h2>
+          Glad to see
+        </h2>
+        <h2>
+          You!
+        </h2>
```

Comment:
## Optional
- Please You can use one "h2" and "br" instead. Example:  "h2" Glad to see "br" You! "h2"

## Important: 
Please do not touch the code when it is in the review process. I am saying this because You made changes while I was reviewing. You can touch different branches but not this one. Not a good practice.

Reply:
Done . And thanks For the help and review. 👍